install-rhel-deps-build-openjdk11.sh

```diff
@@ -117,6 +117,8 @@ TARBALL_VERSION="\${UPDATE}_\${BUILD}\${EA_SUFFIX}"
 PLATFORM_VERSION="\${PLATFORM}_\${TARBALL_VERSION}"
 TARBALL_NAME="\${TARBALL_BASE_NAME}-jdk_\${PLATFORM_VERSION}"
 TARBALL_NAME_JRE="\${TARBALL_BASE_NAME}-jre_\${PLATFORM_VERSION}"
+TARBALL_NAME_SHENDOH="\${TARBALL_BASE_NAME}-jdk-shenandoah_\${PLATFORM_VERSION}"
+TARBALL_NAME_SHENDOH_JRE="\${TARBALL_BASE_NAME}-jre-shenandoah_\${PLATFORM_VERSION}"
 TARBALL_NAME_TEST_IMAGE="\${TARBALL_BASE_NAME}-testimage_\${PLATFORM_VERSION}"
```

Comment:
We should perhaps have a testimage artefact ready too for shenandoah. Just in case.